creator-node/src/routes/files.js

```diff
@@ -990,33 +735,14 @@ module.exports = function (app) {
     })
   )
 
-  app.get(
-    '/ipfs_peer_info',
-    handleResponse(async (req, res) => {
-      const ipfs = req.app.get('ipfsAPI')
-      const ipfsIDObj = await getIPFSPeerId(ipfs)
-      if (req.query.caller_ipfs_id) {
-        try {
-          req.logger.info(`Connection to ${req.query.caller_ipfs_id}`)
-          await ipfs.swarm.connect(req.query.caller_ipfs_id)
-        } catch (e) {
-          if (!e.message.includes('dial to self')) {
-            req.logger.error(e)
-          }
-        }
-      }
-      return successResponse(ipfsIDObj)
-    })
-  )
-
   /**
-   * Serve IPFS data hosted by creator node and create download route using query string pattern
+   * Serve data hosted by creator node and create download route using query string pattern
    * `...?filename=<file_name.mp3>`.
    * @param req
    * @param req.query
    * @param {string} req.query.filename filename to set as the content-disposition header
    * @param {boolean} req.query.fromFS whether or not to retrieve directly from the filesystem and
-   * rehydrate IPFS asynchronously
+   * rehydrate IPFS asynchronously (REMOVED -- only here for backwards compatibility until the client updates)
```

Comment:
oh unless you meant the route itself. let's update the comment to "this route is only here ... " :)

Reply:
I actually just meant that the `fromFS` queryparam is removed because I didn't want to change it if the client is using it. after looking more tho, the client uses the route but not that queryparam so it should be safe to remove. I'll push a commit to remove the queryparam and make a note about the route name